README.md

```diff
@@ -278,6 +278,8 @@ var locationStore = alt.createStore(LocationStore)
 locationStore.myPublicMethod()
 ```
 
+Another explicit, less magical, alternative to setting a method as static is to [declare](http://goatslacker.github.io/alt/docs/createStore/#storemodelexportpublicmethods) `exportPublicMethods` in the constructor.
```

Comment:
Actually we can replace this whole section with `exportPublicMethods` and how to use it.